Havit.Blazor.Components.Web.Bootstrap/Forms/Autosuggests/Internal/HxAutosuggestItems.razor.css

```diff
@@ -0,0 +1,4 @@
+﻿::deep .hx-autosuggest-items-item-focus {
+    background-color: #e9ecef !important;
+    outline: -webkit-focus-ring-color auto 1px !important;
+}
```

Comment:
Please consult with @crdo.
This seems to explicit to me, we should use CSS variable for color (preferably existing one from BS) etc.

Reply:
@crdo The color has been replaced with a Bootstrap CSS variable and the outline removed (to make identifying focused element clearer).